pip/tests/test_qasm.py

```diff
@@ -0,0 +1,808 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from math import pi
+from textwrap import dedent
+import pytest
+from qsharp import (
+    init,
+    TargetProfile,
+    set_quantum_seed,
+    BitFlipNoise,
+    QSharpError,
+    Result,
+)
+from qsharp.estimator import EstimatorParams, QubitParams, QECScheme, LogicalCounts
+from qsharp.interop.qasm import (
```

Comment:
It seems a bit long-winded having `interop` in the module path, and makes QASM sound a bit like an afterthought bolted on with a shim layer. Would just `qsharp.qasm` be feasible? Simpler, shorter, and makes QASM sound more like it has "first class" support in the package.

Reply:
@billti Do you want that for Qiskit as well?

Reply:
Would that be a breaking change? It would look cleaner naming-wise, though Qiskit does still feel like an 'interop' scenario, whereas with the recent work QASM is more directly supported in the package. I could go either way...

Reply:
I can use a redirction shim which would report a warning for any user importing it while automaticatically re-exporting everything from the new location. Then later we could delete the redirection shim.